source/whats-new.txt

```diff
@@ -27,38 +27,41 @@ Learn what's new in:
 
 .. _upcoming-breaking-changes:
 
+Upcoming Breaking Changes
+-------------------------
+
+- `MapReduceIterable <{+api+}/apidocs/mongodb-driver-sync/com/mongodb/client/MapReduceIterable.html>`__
+  and map-reduce methods on `MongoCollection <{+api+}/apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#mapReduce(com.mongodb.client.ClientSession,java.lang.String,java.lang.String)>`__
+  are deprecated in MongoDB v4.2 and later,  and will be replaced by the 
+  aggregation framework.
```

Comment:
This change only applies to v4.4 of the driver and later and should be left out of the cherry-picks for prior versions (v4.0-v4.3)